packages/slides-frontend/src/containers/redux-store/DeckReducer/reducer.ts

```diff
@@ -37,6 +37,43 @@ const DeckState = (state: Deck = initialDeck, action: Action): Deck =>
           ID: uuidv4(),
           itemsArray: [], // add a box that can be image or text
         }
+
+        // this code is to add two text Boxes by default in any new slide
+        // START
```

Comment:
That could be extracted to a separate method/function.

Reply:
:+1: